manticore/platforms/evm.py

```diff
@@ -2164,14 +2181,15 @@ def CALL_gas(self, wanted_gas, address, value, in_offset, in_size, out_offset, o
         )
         fee += self._get_memfee(in_offset, in_size)
 
-        exception = False
         available_gas = self._gas
         available_gas -= fee
 
         exception = Operators.OR(
             Operators.UGT(fee, self._gas),
             Operators.ULT(self.safe_mul(available_gas, 63), available_gas),
         )
+        self.fail_if(exception)
```

Comment:
Does this line need to check the value of `consts.gas` before deciding to raise the exception?